ethstorage/encoder/optimism_encoder.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2022-2023, EthStorage.
+// For license information, see https://github.com/ethstorage/es-node/blob/main/LICENSE
+
+package encoder
+
+// https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/blob.go
```

Comment:
change to a specific version of code? instead of the develop branch? like https://github.com/ethereum-optimism/optimism/blob/v1.6.1/op-service/eth/blob.go